<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPaletteSection</class>
 <widget class="QDialog" name="ExportPaletteSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Palette Section</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>From</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="from000">
          <property name="text">
           <string>0</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="from064">
          <property name="text">
           <string>64</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="from128">
          <property name="text">
           <string>128</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="from192">
          <property name="text">
           <string>192</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="from016">
          <property name="text">
           <string>16</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="from080">
          <property name="text">
           <string>80</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="from144">
          <property name="text">
           <string>144</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="from208">
          <property name="text">
           <string>208</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="from032">
          <property name="text">
           <string>32</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="from096">
          <property name="text">
           <string>96</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="from160">
          <property name="text">
           <string>160</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QRadioButton" name="from224">
          <property name="text">
           <string>224</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="from048">
          <property name="text">
           <string>48</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="from112">
          <property name="text">
           <string>112</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="from176">
          <property name="text">
           <string>176</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QRadioButton" name="from240">
          <property name="text">
           <string>240</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Colors</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="n016">
          <property name="text">
           <string>16</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="n080">
          <property name="text">
           <string>80</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="n144">
          <property name="text">
           <string>144</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="n208">
          <property name="text">
           <string>208</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="n032">
          <property name="text">
           <string>32</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="n096">
          <property name="text">
           <string>96</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="n160">
          <property name="text">
           <string>160</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="n224">
          <property name="text">
           <string>224</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="n048">
          <property name="text">
           <string>48</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="n112">
          <property name="text">
           <string>112</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="n176">
          <property name="text">
           <string>176</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QRadioButton" name="n240">
          <property name="text">
           <string>240</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="n064">
          <property name="text">
           <string>64</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="n128">
          <property name="text">
           <string>128</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="n192">
          <property name="text">
           <string>192</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QRadioButton" name="n256">
          <property name="text">
           <string>256</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportPaletteSection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportPaletteSection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
